### ISSUE TITLE
Question - Is there a way to add functions that are global only to a given context or environment?

### ISSUE THREAD

**Issue by sykesd** [jaccomoc/jactl#80]
We are evaluating Jactl for an end-user/customization programming language in a piece of software.

When adding global functions and members, is there a way to do it without them being available to every single script that Jactl executes?

If you can imagine there are two parts of the system, module A and module B, unrelated to each other, but within the same process that allow customization, it would be great to be able to add in functions and methods as the integration points in such a way that functions/methods from module A are not accessible when executing scripts related to module B.

Looking into the source code this does not look to be possible, but I am wondering if I am missing something.

**Comment by jaccomoc** [comment]
You are right. It is not currently possible but it seems like a reasonable thing to support so I will look into it. Shouldn't be too hard to implement I think.

**Comment by jaccomoc** [comment]
Functions/methods can now be registered against different JactlContexts to allow different sets of scripts to see different sets of custom functions (see #82).
A new Jactl 2.2.0 release has been generated that includes this enhancement.

**Comment by sykesd** [comment]
Wow. Thanks heaps.
